lib/elixir_jobs/offers/offer.ex

```diff
@@ -28,7 +29,7 @@ defmodule ElixirJobs.Offers.Offer do
   end
 
   @required_attrs [:title, :company, :description, :location, :url, :job_place, :job_type]
-  @optional_attrs [:published_at, :slug]
+  @optional_attrs [:published_at, :slug, :summary]
```

Comment:
I would make this mandatory, as it's used in listings. Also, It would be interesting to use it separately from the description, as described in other comment.